load_testing/requirements.txt

```diff
@@ -1,35 +1,35 @@
-backports.functools-lru-cache==1.2.1
-beautifulsoup4==4.4.1
-configparser==3.5.0b2
-gevent==1.1.0
-greenlet==0.4.9
+backports.functools-lru-cache==1.3
+beautifulsoup4==4.5.3
+configparser==3.5.0
+gevent==1.1.2
+greenlet==0.4.11
 inflect==0.2.5
-irc==12.3
+irc==15.0.4
 jaraco.apt==1.0
-jaraco.classes==1.3
-jaraco.collections==1.3.2
+jaraco.classes==1.4
+jaraco.collections==1.5
 jaraco.context==1.5.2
-jaraco.functools==1.9
-jaraco.itertools==1.7.1
-jaraco.logging==1.3.1
-jaraco.text==1.6.3
+jaraco.functools==1.15.1
+jaraco.itertools==2.0
+jaraco.logging==1.5
+jaraco.text==1.8
 jaraco.timing==1.2.2
 konfig==0.9
 loads==0.3
-more-itertools==2.2
+more-itertools==2.3
 Paste==2.0.3
-psutil==4.1.0
-pytz==2016.3
-pyzmq==15.2.0
+psutil==5.0.0
+pytz==2016.10
+pyzmq==16.0.1
 requests==2.9.1
 six==1.10.0
-tempora==1.4
+tempora==1.6.1
 ujson==1.35
-waitress==0.8.10
-WebOb==1.6.0
-WebTest==2.0.20
-ws4py==0.3.4
+waitress==1.0.2
+WebOb==1.7.1
+WebTest==2.0.25
+ws4py==0.3.5
 WSGIProxy==0.2.2
 WSGIProxy2==0.4.2
 yg.lockfile==2.1.1
-zc.lockfile==1.1.0
+zc.lockfile==1.2.1
```

Comment:
This part was necessary for this PR? Could have been it's own PR, but I suppose the git gymnastics are too much. Okay, approved. One thing I would suggest is that Assembl developers update the docs, showing what tools/services they use for managing their PyPi and NPM dependencies. What do you think?